scripts/build_and_push_images.sh

```diff
@@ -21,34 +21,54 @@ INDEX_IMG="$REPO/$OP_NAME-index:$VERSION"
 
 BUNDLE_IMG="$REPO/$OP_NAME-bundle:$VERSION"
 
+AGENT_IMAGE="$OP_NAME-agent"
+AGENT_IMG_BASE="$REPO/$AGENT_IMAGE"
+AGENT_IMG="$AGENT_IMG_BASE:$VERSION"
+
+CLUSTER_BUNDLE_FILE="bundle/manifests/osp-director-operator.clusterserviceversion.yaml"
+
 # Base operator image
 make manager
 make manifests
 make generate
-IMG=${IMG} make docker-build docker-push
+make IMG=${IMG} docker-build docker-push
+
+# Agent image
+make IMG=${AGENT_IMG} DOCKERFILE="Dockerfile.agent" docker-build docker-push
```

Comment:
Just curious: we've previously placed env vars before the command as opposed to after it as args (such as in line 44).  This format works as well?

Reply:
Yes. IIIRC the difference is that env vars will not override the var set in the Makefile.